library/asb_encode_binding.py

```diff
@@ -44,9 +44,18 @@
 
 import json
 import base64
+import os
 from ansible.module_utils.basic import AnsibleModule
+try:
+    from kubernetes import client, config
+    config.load_kube_config()
+    api = client.CoreV1Api()
+except Exception as error:
+    ansible_module.fail_json(msg="Error attempting to load kubernetes client: " + str(error))
 
 ENCODED_BINDING_PATH = "/var/tmp/bind-creds"
+ENV_NAMESPACE = "POD_NAMESPACE"
```

Comment:
why is this set to `POD_NAMESPACE`?

Reply:
https://github.com/openshift/ansible-service-broker/blob/master/pkg/apb/executor.go#L137